packages/element-web-module-api/element-web-module-api.api.md

```diff
@@ -347,6 +347,10 @@ export interface RoomListStoreApi {
 
 // @alpha
 export interface RoomViewProps {
+    // (undocumented)
+    hideComposer?: boolean;
+    // (undocumented)
+    hideHeader?: boolean;

```

Comment:
It doesn't keep the doc which seems the case for all the stuff in this generated file